src/main/java/Main.java

```diff
@@ -2,7 +2,7 @@
 import java.net.ServerSocket;
 
 public class Main {
-    public static void main(String args[]) throws IOException {
+    public static void main(String[] args) throws IOException {
```

Comment:
this is interesting, did it not throw before? fascinating.

Reply:
Apparently the former is C-style so it tolerates it, still seems odd to me.